.rat-excludes

```diff
@@ -109,6 +109,9 @@ TCPIPV6_os.*
 .ism*
 resource.h*
 resource.hm*
+odb.sln
+odb.vcxporj*
```

Comment:
Thank you.

I used to want to add "odb.vcxproj*". However I found the next line "odb.vcxproj.*" was enough . So I remove it.